config/manager/kustomization.yaml

```diff
@@ -14,4 +14,4 @@ patchesStrategicMerge:
 - env.yaml
 images:
 - name: controller
-  newName: quay.io/akaris/controller
+  newName: quay.io/mmahmoud/controller
```

Comment:
That file constantly changes. Just for the future, we might want to check if we add it to gitignore or something

Reply:
yeah we need to add that to the .gitignore or not commit it